processor/samplingprocessor/probabilisticsamplerprocessor/probabilisticsampler.go

```diff
@@ -82,6 +85,16 @@ func (tsp *tracesamplerprocessor) ConsumeTraces(ctx context.Context, td pdata.Tr
 	return tsp.nextConsumer.ConsumeTraces(ctx, sampledTraceData)
 }
 
+func (tsp *tracesamplerprocessor) updateSamplingProbability(sampledSpanAttributes pdata.AttributeMap) {
+	samplingProbability := tsp.samplingProbability
+	attr, found := sampledSpanAttributes.Get(conventions.AttributeSamplingProbability)
+	if found && attr.Type() == pdata.AttributeValueDOUBLE {
+		samplingProbability *= attr.DoubleVal()
```

Comment:
Is this really a multiplication here? I think it depends on the algorithm used, may be a min or multiplication.

Reply:
I think this problem should be somewhat mitigated by this: https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/samplingprocessor/probabilisticsamplerprocessor/probabilisticsampler.go#L110-L115